openpype/hosts/blender/api/lib.py

```diff
@@ -134,6 +134,16 @@ def append_user_scripts():
         traceback.print_exc()
 
 
+def set_app_templates_path():
+    # Blender requires the app templates to be in `BLENDER_USER_SCRIPTS`.
+    # After running Blender, we set that variable to our custom path, so
+    # that the user can use their custom app templates.
+    app_templates_path = os.environ.get("OPENPYPE_APP_TEMPLATES_PATH")
```

Comment:
Maybe we could use first path from `OPENPYPE_BLENDER_USER_SCRIPTS` instead of new env variable?

Reply:
And maybe first path that have existing subfolders `/startup/bl_app_templates_user` (from documentation that is requirement for app templates...)?